lms/static/sass/views/_bookmarks.scss

```diff
@@ -124,4 +125,71 @@
 
 .bookmarks-empty-detail {
   @extend %t-copy-sub1;
-}
\ No newline at end of file
+}
+
+
+// Rules for bookmark icon shown on each sequence nav item
+i.bookmarked {
+  top: -3px;
+  position: absolute;
+  left: 10px;
+}
+
+
+// Rules for bookmark button's different styles
+.bookmark-button-wrapper {
+  text-align: right;
+  margin-bottom: 10px;
+}
+
+@mixin base-style($border-color, $content-color) {
+  background: none;
+  border: 1px solid $border-color;
+  border-radius: ($baseline/4);
+  color: $content-color;
+
+  &:focus, &:active {
+    box-shadow: none;
+  }
+}
+
+@mixin icon-style($content, $color) {
+  &:before {
+    content: $content;
+    font-family: FontAwesome;
+    color: $color;
+  }
+}
+
+@mixin hover-style($border-color, $content-color, $icon-content) {
+  &:hover {
+    background: none;
+    border: 1px solid $border-color;
+    color: $content-color;
+    @include icon-style($icon-content, $content-color);
+  }
+}
+
+.bookmark-button.bookmarked {
+  @include base-style($m-blue, $m-blue);
+  @include icon-style("\f02e", $m-blue);
+  @include hover-style($light-gray, $black, "\f097");
+
+  &:focus {
```

Comment:
The focus styling suggested here is emulating the styles of the opposite state /action of the button here. 

So, in lines 178 -181, a bookmarked button in focus looks like it is not active, and with lines 190 -194, an unbookmarked button when focused looks like it is active. If these lines are removed, we should still be inheriting the outline: blue styling that exists for buttons with focus, which should be enough for accessibility purposes. Does that make sense?